sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/jms/ServiceBusJmsAutoConfiguration.java

```diff
@@ -42,11 +44,18 @@
 @AutoConfigureAfter({
     JndiConnectionFactoryAutoConfiguration.class,
     AzureServiceBusResourceManagerAutoConfiguration.class })
+@ConditionalOnProperty(value = "spring.jms.servicebus.enabled", matchIfMissing = true)
 @ConditionalOnClass({ ConnectionFactory.class, JmsConnectionFactory.class, JmsTemplate.class })
-@EnableConfigurationProperties({ AzureServiceBusJmsProperties.class, JmsProperties.class })
+@EnableConfigurationProperties(JmsProperties.class)
 @Import({ ServiceBusJmsConnectionFactoryConfiguration.class, ServiceBusJmsContainerConfiguration.class })
 public class ServiceBusJmsAutoConfiguration {
 
+    @ConfigurationProperties(prefix = AzureServiceBusJmsProperties.PREFIX)
+    @Bean
+    AzureServiceBusJmsProperties azureServiceBusJmsProperties() {
+        return new AzureServiceBusJmsProperties();
+    }
```

Comment:
What's the difference of writing it here and putting it in the `@EnableConfigurationProperties`?

Reply:
reverted.